packages/web-ui/src/ds/atoms/Tooltip/index.tsx

```diff
@@ -85,9 +86,7 @@ function Tooltip({
       delayDuration={delayDuration}
       disableHoverableContent={disableHoverableContent}
     >
-      <TooltipTrigger asChild onClick={(e) => e.preventDefault()}>
```

Comment:
this prevent default is messing with links, not sure why it's there and there's no comment explaining why so i'm just removing it